src/store/testState.ts

```diff
@@ -143,7 +140,8 @@ const initialState: IState = {
   // Where does the user sit
   userSeat: "player1",
   // The player that won the game
-  winners: [undefined]
+  winners: [undefined],
+  withdrawAddressList: []
```

Comment:
What about this line? Is this related?

Reply:
ah, the linter or smth was complaining that this is missing in that file :)